rotkehlchen/api/rest.py

```diff
@@ -4364,3 +4369,43 @@ def get_evm_counterparties_details(self) -> Response:
             )),
             status_code=HTTPStatus.OK,
         )
+
+    @async_api_call()
+    def invalidate_general_cache(self, cache_type: CacheInvalidationType) -> dict[str, Any]:
+        eth_node_inquirer = self.rotkehlchen.chains_aggregator.ethereum.node_inquirer
+        if cache_type == CacheInvalidationType.CURVE:
```

Comment:
from the conversation in discord he said to keep it simple and I understand that this is to not mess with any special distinction between the different protocols. So a query to this endpoint will query everything. If you understood something different let's clarify it